admin/api/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>admin</artifactId>
+        <groupId>org.codice.ddf.admin.beta</groupId>
+        <version>2.10.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>admin-api</artifactId>
+    <packaging>bundle</packaging>
+
+    <name>DDF :: Admin (Beta) :: API</name>
+    <dependencies>
+
+        <!--todo this dependency needs to be removed once the dependency on the PolicyManager class is fixed-->
```

Comment:
This will need to be a separate ticket, it's the same one being referred to at https://github.com/codice/ddf/pull/1591/files#diff-dc4158b476bd8afa15335cc5e8f2d241R47